sources/models/shared-defaults/nvidia-k8s-device-plugin.toml

```diff
@@ -0,0 +1,15 @@
+# nvidia device plugin service
+[services.nvidia-device-plugin]
+restart-commands = ["/bin/systemctl try-reload-or-restart nvidia-k8s-device-plugin.service"]
+configuration-files = ["nvidia-device-plugin"]
+
+[configuration-files.nvidia-device-plugin]
+path = "/etc/nvidia-k8s-device-plugin/settings.yaml"
+template-path = "/usr/share/templates/nvidia-k8s-device-plugin-conf"
+
+[metadata.settings.kubernetes.nvidia.device-plugin]
+affected-services = ["nvidia-device-plugin"]
+
+[settings.kubernetes.nvidia.device-plugin]
```

Comment:
Does this mean that by default, we don't want to share GPUs? If so, a comment can help.